feat: support custom request headers from java client and migrations tool (#8787)
confluentinc/ksql: docs/operate-and-deploy/migrations-tool.md

some of the statements will have been run on the ksqlDB server while later statements 
have not.

You can optionally pass custom request headers to be sent with all ksqlDB requests 

> made as part of the `apply` command. In order to do so, pass the location of a

Reviewer: ```suggestion
made as part of the `apply` command by passing the location of a
```
